appdata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+    <id>io.github.Faithlife-Community.OuDedetai</id>
+
```

Comment:
this is missing a lot of tags. You can go through and start adding them one by one https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines#id